server/translations.ts

```diff
@@ -29,10 +29,10 @@ function getAllPopularityValues() {
   return allPopularityValues;
 }
 
-function replaceSepPerOS(slug) {
+function replaceSepPerOS(slug: string) {
   if (path.sep !== "/") {
     // In other words, we're on Windows
-    return slug.replaceAll("/", "\\\\");
+    return slug.replace(/\//g, "\\\\");
```

Comment:
Node has `replaceAll` support from v15 onwards (according to BCD), is there some other reason for this change?

Reply:
Yes: https://github.com/mdn/yari/actions/runs/4083183214

Reply:
@LeoMcA Let me know if you'd prefer a different solution, but I gathered the simplest way to avoid the warning is to just use `.replace()` until we bump the target library value.

Reply:
Yep, that's fair